### DISCUSSION TITLE
Dark details on bright model are poorly visible compared to Metashape.

### DISCUSSION THREAD

**Discussion by ETHEntColl** [google/model-viewer#4831] [Q&A]
We're currently exporting glb files from Metashape for use in model-viewer. However on some bright models darker details are often poorly visible. In our case mostly black hair on otherwise pale insects.
This is how the model looks in Metashape:
![metashape](https://github.com/google/model-viewer/assets/53223757/079c0ac2-6add-4c65-aaff-db9a1c4497b7)
and this is how it looks like in model-viewer:
![model-viewer](https://github.com/google/model-viewer/assets/53223757/4650d4f0-c0bc-40f9-8d7d-6f2a978fa949)
Notice how the leg hair is rendered differently. Here is a close-up:
![leg_metashape](https://github.com/google/model-viewer/assets/53223757/da95463c-730c-4dcf-b434-1dae9451cd37)
![leg_modelviewer](https://github.com/google/model-viewer/assets/53223757/05c2ad1d-8b38-42fd-bd97-cecc2d2b27a0)

Is it possible to achieve a similar result in model-viewer? I've tried different tone mapping and playing around with the exposure but not with the desired results. Changing metallic and roughness doesn't seem to have an effect on it either. 
I tested this on the online [model-viewer editor](https://modelviewer.dev/editor/).

Thank you!

**Comment by elalish** [comment]
It could be that Metashape has disabled tone mapping entirely (linear). This isn't a good idea, but it happens sometimes, especially when a site works with models that are entirely matte, which this appears to be? Can you tell me more about what you're trying to achieve and why these darker hairs are a problem? You can check linear tone mapping here, to see if that's the issue: https://tone-mapping.glitch.me/ Also, what renderer does Metashape use?
